pyaerocom/config.py

```diff
@@ -243,6 +204,24 @@ def __init__(self, config_file=None, try_infer_environment=True):
         self.last_config_file = None
         self._ebas_flag_info = None
 
+        self._outhomename = "MyPyaerocom"
```

Comment:
Maybe it would be useful declaring a `MYPYAEROCOM` somewhere which we import to all place it's needed. Safer from typos and if it ever changes we only need to modify it once.

Reply:
This **is** the only place where the name appears.